File: 1250_Check_If_It_Is_a_Good_Array/solution.c
int GCD(int a, int b) { return b == 0 ? a : GCD(b, a % b); }
bool isGoodArray(int* nums, int numsSize) {
    int gcd = nums[0];
    for (int i = 1; i < numsSize; i++) {
        gcd = GCD(gcd, nums[i]);
    }
    return gcd == 1;
}

File: 1250_Check_If_It_Is_a_Good_Array/solution.py
class Solution:
    def isGoodArray(self, nums: List[int]) -> bool:
        def gcd(a: int, b: int) -> int:
            if b == 0:
                return a
            else:
                return gcd(b, a % b)

        GCD = nums[0]
        for i in range(len(nums) - 1):
            GCD = gcd(GCD, nums[i + 1])

        return GCD == 1
